faunadb-scala/src/main/scala/faunadb/query/QueryMacros.scala

```diff
@@ -14,7 +14,7 @@ class QueryMacros(val c: whitebox.Context) {
     val (bindings, expr) = c.untypecheck(block.duplicate) match {
       case Block(stmts, expr) =>
         stmts foreach {
-          case v @ ValDef(_, _, _, _) => ()
+          case ValDef(_, _, _, _) => ()
```

Comment:
We could even just match on the type, right? :)

```scala
case _: ValDef => ()
```

Reply:
https://github.com/fauna/faunadb-jvm/pull/245/commits/8e95c1e9bf7f21435725e9a202d8889a3c1398d9